services/common/src/components/projectSummary/ProjectManagement.tsx

```diff
@@ -0,0 +1,195 @@
+              required
+              component={RenderSelect}
+              data={projectSummaryStatusCodes}
+            />
+          )}
+          <Alert
+            message={projectSummaryStatusCodes.find((code) => code.value === status_code).label}
+            type="warning"
+            showIcon
+            className="margin-large--bottom"
+          />
+          <Row gutter={8} justify="start" align="middle">
+            <Col>
+              <Title level={4}>Assigned Project Lead</Title>
+            </Col>
+            <Col>
+              <Paragraph className="margin-medium--bottom">(Optional)</Paragraph>
+            </Col>
+          </Row>
+          <Field
+            showOptional={false}
+            id="project_lead_party_guid"
+            name="project_lead_party_guid"
+            label="Select the Project Lead responsible for this project. This person will serve as the main
+        contact for mine proponents and the Ministry of Environment."
+            component={RenderSelect}
+            data={projectLeadData}
+          />
+          {!isNewProject && !isProjectLeadAssigned && (
+            <Alert
+              message="This project does not have a Project Lead"
+              description={<p>Please assign a Project Lead to this project.</p>}
+              type="warning"
+              showIcon
+            />
+          )}
+          {isCore && (
+            <div className="margin-large--top">
+              <Row gutter={12.5} align="middle" className="margin-medium--bottom">
+                <Col>
+                  <FontAwesomeIcon size="lg" icon={faLock} className="color-primary" />
+                </Col>
+                <Col>
+                  <Title level={3} className="color-primary margin-none">
+                    Internal Ministry Comments
+                  </Title>
+                </Col>
+              </Row>
+              <Title level={4} className="margin-small--bottom">
+                Internal Ministry Comment
+              </Title>
+              <Paragraph>
+                Add comments to this project description for future reference. Anything written in
+                these comments may be requested under FOIPPA. Keep it professional and concise.
+              </Paragraph>
+              <div className="project-summary-ministry-comments">
+                <MinistryCommentPanel
+                  renderEditor={true}
+                  onSubmit={submitComment}
+                  loading={false}
+                  maxLength={500}
+                  comments={ministryComments?.map((comment: IProjectSummaryMinistryComment) => ({
+                    key: comment.project_summary_ministry_comment_guid,
+                    author: comment.update_user,
+                    content: comment.content,
+                    actions: null,
+                    datetime: comment.update_timestamp,
+                  }))}
+                  createPermission={Permission.EDIT_PROJECT_SUMMARIES}
```

Comment:
So, I think the weird import from core-web instead of common for Permission is because I've resisted bringing the Permission file into common as it's essentially duplicating USER_ROLES. If you give it USER_ROLES.role_edit_project_summaries instead, you can import from common. `userHasRole` in authenticationReducer has been setup to take either USER_ROLES.x or Permission.x in order to be accommodating.